cmd/node/config/config.toml

```diff
@@ -350,6 +350,12 @@
     Type = "SizeLRU"
     SizeInBytes = 104857600 #100MB
 
+[TrieNodesChunksDataPool]
+    Name = "TrieNodesDataPool"
```

Comment:
There is another data pool named "TrieNodesDataPool". Is this intentional?

Reply:
yes, the first one will keep the complete trie nodes, this one will keep the partial trie nodes along with the chunk information